containers/ruby-consul-terraform/Dockerfile-fresh

```diff
@@ -0,0 +1,58 @@
+FROM alpine:edge
```

Comment:
The name of this file is pretty ambiguous. Can you rename it to something more specific, and if it's replacing something can you delete the thing it's replacing?